src/language-css/parser-postcss.js

```diff
@@ -17,11 +17,8 @@ function parseSelector(selector) {
     };
```

Comment:
regarding this comment above:

```
// If there's a comment inside of a selector, the parser tries to parse
// the content of the comment as selectors which turns it into complete
// garbage. Better to print the whole selector as-is and not try to parse
// and reformat it.
```

This is no longer true. We have many tests around comments to ensure they are preserved across parse and re-stringification.

https://github.com/postcss/postcss-selector-parser/blob/master/src/__tests__/comments.js
https://github.com/postcss/postcss-selector-parser/blob/master/src/__tests__/attributes.js#L355-L394

But... the way you are using the selector api causes the comments to be dropped in some cases. Comments are stored in the `raws`. There are methods for accessing the value(s) of a node and the spaces around them in a consistent way. If you think we need other methods, feel free to suggest or send a PR.

Reply:
If I remember correctly, people were opening issues about `//`-style comments, and I don’t see any tests for those after a quick glance through the links you provided.

Reply:
No, it was about regular CSS comments. See #3649

Reply:
Supporting `//` style comments isn't that hard. if it's really important I can add support to our lexer for them.